pkg/lib/operation/project/remote/file/download/operation.go

```diff
@@ -2,20 +2,27 @@
 package download
 
 import (
```

Comment:
V tomto subore je to cele implementovane.

Povodne to riesili J+H, islo to, su k tomu aj 4 E2E testy, no par veci tu nebolo doriesenych a ukazalo sa to az pri stahovani vacsich suborov.

Reply:
Asi je jednoduchsie pozriet cely subor odznova: https://github.com/keboola/keboola-as-code/blob/39fb26601eedcf4309b57cea829f209d6167fe60/pkg/lib/operation/project/remote/file/download/operation.go